Ban unreachable nodes on CreateSession failures (#2204)
ydb-platform/ydb-go-sdk: CHANGELOG.md

> * Added pessimization connection when create table or query session failed with OVERLOADED, UNAVAILABLE, or client-side context.DeadlineExceeded

Author: ```suggestion
* * Added connection pessimization when creating table or query session fails with OVERLOADED, UNAVAILABLE, or client-side context.DeadlineExceeded
```

> * Added connection pessimization when creating table or query session fails with OVERLOADED, UNAVAILABLE, or client-side context.DeadlineExceeded

Author: ```suggestion
* Added connection pessimization when creating table or query session fails with `OVERLOADED`, `UNAVAILABLE`, or client-side `context.DeadlineExceeded`
```

> * * Added connection pessimization when creating table or query session fails with OVERLOADED, UNAVAILABLE, or client-side context.DeadlineExceeded

Author: ```suggestion
* Added connection pessimization when creating table or query session fails with OVERLOADED, UNAVAILABLE, or client-side context.DeadlineExceeded
```
